WooCommerce/Classes/ViewRelated/CardPresentPayments/CardReaderConnectionController.swift

```diff
@@ -209,11 +269,19 @@ private extension CardReaderConnectionController {
                     return
                 }
 
+                /// If we have found multiple readers, advance to foundMultipleReaders
+                ///
+                if self.foundReaders.count > 1 {
```

Comment:
A bit inspired by Rails... I'm dead serious here: would it make sense to add a `containsMoreThanOne` property to Array+Helpers? 

I know it's forcing the metaphor a bit, but we are doing this `if count > 1` thing in several places and it could be a nice bit of syntactic sugar

Reply:
Fixed in two places in https://github.com/woocommerce/woocommerce-ios/pull/4931/commits/436a42b73dec32e5ee558fe0a14e9a987806b454